addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -454,6 +452,9 @@ function factory(dependencies) {
             readonly: true,
             required: true,
         }),
+        wasInitThreadConsidered: attr({
+            default: false,
+        }),
```

Comment:
Field looks very off. Name is too obscure. At a glance of code below, this is because the concept is to algorithmically based instead of on a state.